corgi/core/migrations/0082_fix_build_sources_typo.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.18 on 2023-06-19 02:18
+
+import os
+
+from django.db import migrations
+from django.db.models import Value, functions
+
+
+def fix_build_source_urls(apps, schema_editor):
+    """Convert source URLs with two /git/git/ in path to just one /git/ (typo in last migration)"""
+    SoftwareBuild = apps.get_model("core", "SoftwareBuild")
+
+    # Find any build where the source URL has two "/git/git/"
+    # Replace it with just one "/git/"
+    one_git_value = Value("/git/")
+    two_git_path = "/git/git/"
+    two_git_value = Value(two_git_path)
+
+    builds_to_update = SoftwareBuild.objects.filter(source__contains=two_git_path)
+    builds_to_update.update(source=functions.Replace("source", two_git_value, one_git_value))
```

Comment:
I concur that this kind of change is probably better handled manually than with a migration. It's a really specific circumstance that we ourselves created; it doesn't originate in data that we're getting from a third party. That said, since the area of effect of this migration is so small, I think the likelihood of side effects is low.